### ISSUE TITLE
cmd/vet: check %w Errorf flag

### ISSUE THREAD

**Issue by josharian** [golang/go#32070]
We should teach cmd/vet about the new %w Errorf flag.

Vet seems perfectly happy with this code:

```go
func f(s string) error {
	return fmt.Errorf("abc %w", s)
}
```

cc @mvdan @jba @mpvl

**Comment by mvdan** [comment]
Weren't part of the errors changes reverted in master? Just want to double-check that we don't add checks to vet that don't reflect the released behavior of fmt.

**Comment by neild** [comment]
The `%w` format was retained, with some slight modification to make it less of a special case:
https://github.com/golang/go/issues/29934#issuecomment-489682919 (decision)
https://golang.org/cl/176998 (updated `%w` implementation)

**Comment by andybons** [comment]
@neild any update on this?

**Comment by neild** [comment]
https://go-review.googlesource.com/c/tools/+/177601

**Comment by odeke-em** [comment]
Hello @neild, this is a kind ping to check on this as this Jonathan reviewed and +2'd it and the associated CL is marked for Go1.13 which will soon be released. Thank you!

**Comment by logrusorgru** [comment]
So, go1.13, `go vet` doesn't say anything about "%w" in a `Sprintf` for example. Is it expected behavior?

```go
package main

import (
	"errors"
	"fmt"
)

var (
	ErrOne = errors.New("one error")
	ErrTwo = errors.New("two error")
)

func main() {

	var (
		err = fmt.Errorf("double wrap: %w, and %w", ErrOne, ErrTwo)
		str = fmt.Sprintf("double wrap: %w, and %w", ErrOne, ErrTwo)
	)

	fmt.Println(err)
	fmt.Println(str)
}
```

[play.golang.org](https://play.golang.org/p/xlikbh0H_GI)

```
$ go vet main.go
```
```
$ go version
go version go1.13.3 linux/amd64
```

**Comment by josharian** [comment]
This should be fixed for 1.14. See #32070.
